scripts/check_no_panics.py

```
@@ -251,6 +273,221 @@ def is_test_only_path(path: str) -> bool:
+def load_reborn_baseline(
+    path: pathlib.Path,
+) -> collections.Counter[tuple[str, str]]:
+    approved: collections.Counter[tuple[str, str]] = collections.Counter()
+    for line_no, raw in enumerate(path.read_text(encoding="utf-8").splitlines(), 1):
+        if not raw.strip() or raw.lstrip().startswith("#"):
+            continue
+        fields = raw.split("\t", 2)
+        if len(fields) != 3 or not fields[2].strip():
+            raise RuntimeError(
+                f"{path}:{line_no}: expected path<TAB>source<TAB>non-empty reason"
+            )
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)